functions/package.json

```diff
@@ -3,16 +3,17 @@
   "description": "Cloud Functions for Firebase",
   "scripts": {
     "lint": "",
-    "serve": "firebase emulators:start --only functions",
-    "shell": "firebase functions:shell",
+    "serve": "npm run build && firebase emulators:start",
```

Comment:
recommended to use `npm run serve` from now on to automatically compile with typescript